bundles/config/org.eclipse.smarthome.config.discovery/src/main/java/org/eclipse/smarthome/config/discovery/DiscoveryResultBuilder.java

```diff
@@ -134,7 +134,7 @@ public DiscoveryResultBuilder withTTL(long ttl) {
      */
     public DiscoveryResult build() {
         return new DiscoveryResultImpl(thingTypeUID, thingUID, bridgeUID, properties, representationProperty, label,
-                ttl);
+                ttl, null);
```

Comment:
Is null really the right thing to do?
I would have expected that the builder provides an immutable with this field already set - see my comment in AbstractDiscoveryService.